examples/stable/elasticsearch/elasticsearch-blueprint.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: elasticsearch-blueprint
+actions:
+  backup:
+    type: StatefulSet
+    outputArtifacts:
+      cloudObject:
+        keyValue:
+          backupLocation: "{{ .Phases.backupToObjectStore.Output.backupLocation }}"
+    phases:
+    - func: KubeTask
+      name: backupToObjectStore
+      args:
+        namespace: "{{ .StatefulSet.Namespace }}"
+        image: "kanisterio/es-sidecar:0.20.0"
```

Comment:
Please change the version to `0.21.0` in all 3 places in this blueprint.

Reply:
Updated the sidecar version to `0.21.0`